server/database/models/rating.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable no-unused-vars */
+module.exports = (sequelize, DataTypes) => {
+  const Rating = sequelize.define('Rating', {
+    rating: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    userId: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    articleId: {
+      type: DataTypes.UUID,
+      defaultValue: DataTypes.UUIDV4,
+      allowNull: false,
+      onDelete: 'CASCADE',
+      reference: {
+        model: 'Article',
+        key: 'id',
+      },
+    },
+  }, {});
+
+  Rating.associate = (models) => {
+    // associate rating with article
+    Rating.belongsTo(models.Article, {
+      foreignKey: 'articleId',
+      onDelete: 'CASCADE',
+    });
```

Comment:
Similar to the comment I made above, Is rating not supposed to belong to a user or do you have any reason for not doing this?

Reply:
Ok, thanks for the observation. I think the association that is necessary here is the relationship between Rating and Article. Since there is no need to return the users along with the ratings but rather the article